R/set-record-option.R

```diff
@@ -0,0 +1,110 @@
+
```

Comment:
On my end, styler (1.10.1) removes this blank line.  Please run this through `styler_pkg()` (directly or with `make fmt`).

Reply:
Interesting. I had run it, but maybe it behaved a little differently on my end for whatever reason